admin/src/scenes/classe/components/modale/modaleDelete.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { MdOutlineDangerous } from "react-icons/md";
+import { HiOutlineExclamation } from "react-icons/hi";
+import { ModalConfirmation } from "@snu/ds/admin";
+
+interface Props {
+  modaleDelete: boolean;
+  setModaleDelete: (b: boolean) => void;
+  onDelete: (action: "delete") => void;
```

Comment:
```suggestion
  onDelete: () => void;
```
vu que c'est `onDelete` et pas `onAction` ou `onChange`, le param "delete" n'est pas nécessaire.